src/datepicker/datepicker.component.ts

```diff
@@ -176,8 +179,21 @@ export class DatePicker implements OnDestroy, OnChanges, AfterViewChecked {
 
 	protected flatpickrInstance = null;
 
+	protected scrollSubscription: Subscription = new Subscription();
+
 	constructor(protected elementRef: ElementRef) { }
 
+	ngOnInit() {
```

Comment:
Should be `AfterViewInit` ... the view isn't guaranteed to be instantiated in `OnInit`

Reply:
Done!